webapp/client/src/pages/FilingFailurePage.test.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { I18nextProvider } from "react-i18next";
+import i18n from "i18next";
+import FilingFailurePage from "./FilingFailurePage";
+
+describe("FilingFailurePage", () => {
+  beforeEach(() => {
+    render(<FilingFailurePage />);
+  });
+
+  it("should render red cross icon", () => {
+    expect(screen.getByText("failure_icon.svg").closest("svg")).toHaveAttribute(
+      "alt",
+      expect.stringContaining("Weißes Kreuzzeichen auf rotem Kreis")
+    );
+  });
```

Comment:
This tests two things. We could also split that into two (`should render Red Cross icon` and `should set correct alt text`)

Reply:
True! Done :)